packages/rx_bloc_generator/pubspec.yaml

```diff
@@ -10,19 +10,19 @@ environment:
   sdk: ">=3.0.5 <4.0.0"
 
 dependencies:
-  analyzer: ^6.0.0
+  analyzer: ^7.0.0
```

Comment:
I guess that we need to apply the >=5.2.0 <8.0.0 constraint to ensure the other packages such as go_router_builder can work along with the rx_bloc_generator

https://github.com/Prime-Holding/rx_bloc/actions/runs/12434076485/job/34716986362?pr=928

<img width="1281" alt="image" src="https://github.com/user-attachments/assets/121b2968-0bb1-487d-9949-b4f641d031c8" />